src/main/java/org/openelisglobal/resultlimit/service/ResultLimitService.java

```diff
@@ -45,4 +45,13 @@ public interface ResultLimitService extends BaseObjectService<ResultLimit, Strin
     List<ResultLimit> getResultLimits(Test test);
 
     ResultLimit getResultLimitForAnalysis(Analysis analysis);
+
+    List<? extends Number> getHighRangeHigh();
```

Comment:
getHighRangeHigh, getHighRangeLow, getLowRangeHigh, and getLowRangeLow I would expect these to take a resultLimit object to get the configured values for a single resultLimit, not the entire list of ranges for all result limits, but that is also unnecessary as we can get the value directly from the resultLimit then